backend/src/main/java/com/crm/exception/ErrorResponse.java

```diff
@@ -2,13 +2,15 @@
 
 import lombok.AllArgsConstructor;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 import java.util.List;
 
 @Getter
 @Setter
 @AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Chyba nie jest to potrzebne

Reply:
Usunięte

Reply:
Niestety ten bezargumentowy konstruktor jest potrzebny. W moich testach integracyjnych mapuję odpowiedzi do ErrorResponse i brak tej adnotacji powoduje wyrzucenie błędu, o braku konstruktora domyślnego.